server/src-rsr/initialise.sql

```diff
@@ -294,7 +295,8 @@ CREATE TABLE hdb_catalog.event_log
   created_at TIMESTAMP DEFAULT NOW(),
   locked BOOLEAN NOT NULL DEFAULT FALSE,
   next_retry_at TIMESTAMP,
-  archived BOOLEAN NOT NULL DEFAULT FALSE
+  archived BOOLEAN NOT NULL DEFAULT FALSE,
+  paused BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
It seems quite curious to me that each `event_log` record has a `paused` field—you don’t pause an event, you pause a trigger! When a trigger is paused, it shouldn’t create any more events at all, not “paused events.”

I’d remove the `paused` field from `event_log` and change `insert_event_log` so that it just returns without doing anything if the event is paused. Then you don’t have to worry about filtering out paused events in `fetchEvents`.

Reply:
@lexi-lambda I was creating the event log events with paused as one of the columns because I thought it would be a good idea to track the events that were tracked when the trigger was paused, maybe I over-thought it 😅 .